circuits/bin/dkg/e_sm_share_computation/README.md

```
@@ -1 +1,52 @@
+
+    style Focus fill:#5B9BD5,stroke:#2E75B6,stroke-width:3px
+    style Input1 fill:#000000,stroke:#999,stroke-width:2px,stroke-dasharray: 5 5
+    style Output1 fill:#000000,stroke:#999,stroke-width:2px,stroke-dasharray: 5 5
+    style Output2 fill:#000000,stroke:#999,stroke-width:2px,stroke-dasharray: 5 5
+
+    linkStyle 0 stroke:#808080,stroke-width:2px
+    linkStyle 1 stroke:#808080,stroke-width:2px
+    linkStyle 2 stroke:#808080,stroke-width:2px
+```
+
+### Metadata
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)